lib/daru/dataframe.rb

```diff
@@ -2006,6 +2005,18 @@ def split_by_category cat_name
 
     private
 
+    def headers
+      if index.is_a?(MultiIndex)
+        Daru::Index.new(index.name) | @vectors
+      else
+        Daru::Index.new([index.name]) | @vectors
```

Comment:
Adding the index name(s) at the front of vectors, to display as headers.
E.g. `index_name       a        b`